.coveragerc

```diff
@@ -2,14 +2,19 @@
 omit =
     urllib3/packages/*
     urllib3/contrib/*
+    test/*
```

Comment:
I found these useful when combining and reporting coverage data, otherwise the final report would include many undesirable packages.